store/imagestore/remote.go

```diff
@@ -48,23 +53,26 @@ func remoteRowScan(rows *sql.Rows, remote *Remote) error {
 
 // GetRemote tries to retrieve a remote with the given aciURL. found will be
 // false if remote doesn't exist.
-func GetRemote(tx *sql.Tx, aciURL string) (remote *Remote, found bool, err error) {
-	remote = &Remote{}
+func GetRemote(tx *sql.Tx, aciURL string) (*Remote, error) {
```

Comment:
Godoc needs to be fixed with the new NotFound type (nil or error, see above comment)